docs/content/users/quickstart.md

```diff
@@ -769,6 +750,29 @@ The Laravel project type can be used for [StarterKits](https://laravel.com/docs/
 
     The [`server.js`](https://github.com/ddev/test-nodejs/blob/main/server.js) used here is a trivial Express-based Node.js webserver. Yours will be more extensive.
 
+## OpenMage
+
+=== "Git Clone"
+
+    ```bash
+    mkdir my-openmage-site && cd my-openmage-site
+    git clone https://github.com/OpenMage/magento-lts .
+    ddev config --project-type=magento --php-version=8.1 --web-environment-add=MAGE_IS_DEVELOPER_MODE=1
```

Comment:
Why are we using an old version of PHP here? I do see that suggestion in https://docs.openmage.org/users/requirements/ but am surprised that they'd be that far back. DDEV's default is currently 8.3, and will change to 8.4 in the fall. If this is OK and recommended, it's OK, but I was wondering if maybe the OpenMage doc was out of date.

Reply:
Thanks. DOCS are not up 2 date. Current OM release supports php74, but with next release we require php81

https://github.com/OpenMage/magento-lts/pull/4124 (not released yet)

Moving to php83 is was planned for upcoming releases.

Reply:
Thanks, I know you'll update when it supports current stable.